e2e/e2e-windows-scenario.sh

```diff
@@ -109,6 +136,10 @@ az deployment group create --resource-group $MC_RESOURCE_GROUP_NAME \
 retval=$?
 set -e
 
+# delete cse package in storage account
+${array[0]}/azcopy rm "https://${STORAGE_ACCOUNT_NAME}.blob.core.windows.net/${STORAGE_PACKAGE_CONTAINER}/${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip?${tokenWithoutQuote}"
```

Comment:
this step may not be executed for some failures. I think that we have two options:
1. use `--overwrite` in uploading the test cse package
2. check whether a file with the same name exists before uploading the test cse package.

Reply:
I use the random-generated vmss name to specify a cse package. cse packages with the same name may bring the problem when one pipeline wants to use the package while another pipeline wants to delete it.

Reply:
for option 2, we can use a different name when it exists. In most scenarios, it should have no problem if we add timestamp in the name. But it is better to check it before uploading it or check the upload result if it fails because the file exists.

Reply:
Thanks. I will take option 2 and add it.